server.go

```diff
@@ -62,6 +63,10 @@ func (s server) Load(response http.ResponseWriter, request *http.Request) {
 		return
 	}
 
+	if q := request.URL.RawQuery; q != "" && !strings.Contains(url, "?") {
```

Comment:
There are, however the url you redirect to doesn't have to be a valid URL (at least not yet), therefore they won't work.